src/components/phone/phone.js

```diff
@@ -0,0 +1,221 @@
+          },
+      soundClasses: classnames('bg-gray-dark', {
+        'w3 round-l h24': portrait,
+        'h3 round-b w24 none block-mm': landscape
+      }),
+      soundContainerClasses: classnames(
+        'flex-parent absolute flex-parent--space-between-main',
+        {
+          'flex-parent--column h60': portrait,
+          w60: landscape
+        }
+      ),
+      receiverClasses: classnames(
+        'flex-parent bg-gray-dark flex-parent--center-main flex-parent--center-cross',
+        {
+          h36: portrait,
+          w36: landscape
+        }
+      ),
+      screenClasses: classnames(
+        'relative flex-parent flex-parent--center-cross flex-parent--center-main ',
+        {
+          'hmin120-mm hmin60': landscape,
+          hmin300: portrait
+        }
+      ),
+      screenStyles: {
+        overflow: 'hidden',
+        borderRadius: ios ? '18px' : undefined,
+        height: portrait ? `calc(100% - ${ios ? '18px' : '72px'})` : '100%',
+        width: landscape ? (android ? `calc(100% - 36px)` : '100%') : undefined,
+        marginTop: portrait && ios ? '-18px' : undefined,
+        marginLeft: landscape && ios ? '-18px' : undefined
+      }
+    };
+
+    return (
+      <div className={config.containerClasses} style={config.containerStyles}>
+        {/* power button */}
+        <div className={config.powerClasses} style={config.powerStyles} />
+        {/* other android button */}
+        {android && (
+          <div
+            className={config.otherButtonClasses}
+            style={config.otherButtonStyles}
+          />
+        )}
+        {/* sound buttons */}
+        {ios && (
+          <div
+            className={config.soundContainerClasses}
+            style={config.soundStyles}
+          >
+            <div className={config.soundClasses} />
+            <div className={config.soundClasses} />
+          </div>
+        )}
+        {/* notch */}
+        <div
+          className={config.notchContainerClasses}
+          style={config.notchContainerStyles}
+        >
+          <div className={config.notchClasses} style={config.notchStyles}>
+            {ios && (
+              <div
+                className={config.notchCurveLeftClasses}
+                style={config.notchCurveLeftStyles}
+              />
+            )}
+            {/* speaker */}
+            <div className={config.speakerClasses} />
+            {/* camera */}
+            <div className={config.cameraClasses} style={config.cameraStyles} />
+            {/* top notch */}
+            {ios && (
+              <div
+                className={config.notchCurveRightClasses}
+                style={config.notchCurveRightStyles}
+              />
+            )}
+          </div>
+        </div>
+        {/* screen */}
+        <div style={config.screenStyles} className={config.screenClasses}>
+          <div className="loading loading--dark absolute z0" />
+          <div className="relative">{props.children}</div>
```

Comment:
@katydecorah in addition to swapping out the device frames for videos in your test branches, I also tested using this around images (because it would be great for all images and videos to be consistent!).

However, it seems like the logic in `AppropriateImage` might think the container width is 0px, so it's always rendering the smallest image (even when the screen is wide enough that it should be rendering a larger version). After the wrong (too small) image is rendered, if i resize the browser it recalculates and swaps in the right (bigger) version of the image.

I was able to get around this by adding `w-full` to this `div`. In my testing, it didn't seem to have any negative impacts, but I'm curious what you think about this solution!

Reply:
@colleenmcginnis would you mind pushing an example of AppropriateImages to one of the test branches? I haven't been able to recreate this yet 🙏